site/src/Util/Notification.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace App\Util;
+
+use App\Entity\User;
+
+/**
+ * Data class for all Notifications
+ *
+ * @author Aquib Baig <aquibbaig97@gmail.com>
+ */
+class Notification
+{
+    /**
+     * Subject of the Notification
+     *
+     * @var string $subject
+     */
+    protected $subject;
+
+    /**
+     * Actual definition of the Notification
+     *
+     * @var string $message
+     */
+    protected $message;
+
+    /**
+     * User associated with the Notification
+     *
+     * @var User $user
+     */
+    protected $user;
+
+    /**
+     * Who issued the Notification
+     *
+     * @var string author
+     */
+    protected $author;
+
+    /**
+     * The type of Notification
+     *
+     * @var string $type
+     */
+    protected $type;
+
+    /**
+     * When is the Notification created
+     *
+     * @var \DateTime $createdAt
+     */
+    protected $createdAt;
+
+    /**
+     * Notification constructor.
+     * @throws \Exception
```

Comment:
does it?

Reply:
https://php.net/manual/en/datetime.construct.php

It emits an exception in case of error.